docs/en/reference/caching.rst

```diff
@@ -47,4 +47,4 @@ object is closed:
 
 .. warning::
 
-    When using the cache layer not all fetch modes are supported. See the code of the `ResultCacheStatement <https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Cache/ResultCacheStatement.php>`_ for details.
+    When using the cache layer not all fetch modes are supported. See the code of the `ResultCacheStatement <https://github.com/doctrine/dbal/blob/2.10.x/lib/Doctrine/DBAL/Cache/ResultCacheStatement.php>`_ for details.
```

Comment:
I'm not sure about this change. Because your changes done in this branch will be upmerged to the newer versions where 2.10.x isn't the correct url part. We would like to make upmerges as simple as possible without doing further version related adaptations.

@greg0ire Do you have a suggestion on how to handle branch names in the docs between branches?

Reply:
> 2.10.x isn't the correct url part

The `lib/Doctrine/DBAL` part will not be correct either since we moved to `src`. I'd suggest removing the url completely, and using a fully qualified class name instead.

Reply:
Changed to full class name.